config/translations/ru/create.nodes.yml

```diff
@@ -1,29 +1,28 @@
-description: 'Create dummy nodes for your Drupal 8 application.'
-help: 'The <info>create:nodes</info> command helps you generate fake terms.'
-welcome: 'Welcome to the Drupal nodes generator'
+description: 'Создание фиктивных материалов для Drupal 8.'
+help: 'Команда <info>create:nodes</info> поможет вам создать фиктивные материалы.'
+welcome: 'Добро пожаловать в генератор Drupal материалов'
 arguments:
-    content-types: 'Content type(s) to be used in node creation'
+  content-types: 'Тип(ы) содержимого, которые будут использованы при создании материалов'
 options:
-    limit: 'How many nodes would you like to create'
-    title-words: 'Maximum number of words in node titles'
-    time-range: 'How far back in time should the nodes be dated'
+    limit: 'Сколько материалов вы хотите создать'
+    title-words: 'Максимальное количество слов в заголовках материалов'
+    time-range: 'Как далеко назад во времени материалы должны быть датированы'
 questions:
-    content-type: 'Select content type(s) to be used on node creation'
-    limit: 'Enter how many nodes would you like to generate'
-    title-words: 'Enter the maximum number of words in titles'
-    time-range: 'How far back in time should the nodes be dated?'
+    content-type: 'Выберите тип(ы) содержимого, которые будут использованы при создании материалов'
+    limit: 'Введите значение количества материалов, которые вы хотите создать'
+    title-words: 'Введите максимальное количество слов в заголовках'
+    time-range: 'Как далеко назад во времени материалы должны быть датированы?'
     time-ranges:
-        - Now
-        - '1 hour ago'
-        - '1 day ago'
-        - '1 week ago'
-        - '1 month ago'
-        - '1 year ago'
+        - 'Сейчас'
+        - '1 час назад'
+        - '1 день назад'
+        - '1 неделю назад'
+        - '1 месяц назад'
+        - '1 год назад'
 messages:
-    node-id: 'Node Id'
-    content-type: 'Content type'
-    title: Title
-    created: 'Created Time'
-    invalid-content-types: 'Content types %s are invalid'
-    created-nodes: 'Created %s nodes successfully'
-    generated-content: 'Created %s nodes successfully'
+    node-id: 'Идентификатор материала'
+    content-type: 'Тип содержимого'
+    title: 'Заголовок'
+    created: 'Время создания'
+    invalid-content-types: 'Типы содержимого %s недопустимы'
+    created-nodes: '%s материалов создано успешно.'
```

Comment:
generated-content: 'Created %s nodes successfully' was deleted could you restore, please